### PR TITLE
[travis] update "posix" and "posix-ncp-spi" targets to use new configure options

### PR DESCRIPTION
**Pull request by abtink**

### CODE REVIEW

**Comment by jwhui** [comment]
How does this apply to the "posix-app-ncp" target as the title suggests?

**Comment by abtink** [comment]
Ah. I miss-typed. I meant "posix-ncp-spi" target. Let me fix.

File: .travis/script.sh [hunk 1]
```diff
@@ -484,7 +484,8 @@ set -x
     git clean -xfd || die
     ./bootstrap || die
     ./configure                             \
-        --enable-ncp-app=all                \
+        --enable-ncp                        \
+        --enable-ftd                        \
```
Comment (jwhui) [inline .travis/script.sh]: This was previously `all`. Is the omission of `mtd` intentional here?

Comment (abtink) [reply .travis/script.sh]: no. thanks for noticing.

**Pull request merged**